backend/src/main/java/ca/mcgill/purposeful/controller/IdeaController.java

```diff
@@ -93,4 +97,19 @@ public IdeaDTO modifyIdea(
             iconUrlId);
     return new IdeaDTO(modifiedIdea);
   }
+
+  /**
+   * Remove an idea by its id
+   *
+   * @param id the idea's id
+   * @return a response entity with a message instance and the HttpStatus
```

Comment:
Give yourself some credit and add yourself as the author in the javadoc 😉

Reply:
Will do! haha